pkgs/tools/filesystems/fwanalyzer/default.nix

```diff
@@ -0,0 +1,37 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, e2tools
+, mtools
+}:
+
+buildGoModule rec {
+  pname = "fwanalyzer";
+  version = "1.4.3";
+
+  src = fetchFromGitHub {
+    owner = "cruise-automation";
+    repo = pname;
+    rev = version;
+    sha256 = "1pj6s7lzw7490488a30pzvqy2riprfnhb4nzxm6sh2nsp51xalzv";
+  };
+
+  vendorSha256 = "1cjbqx75cspnkx7fgc665q920dsxnsdhqgyiawkvx0i8akczbflw";
+
+  subPackages = [ "cmd/${pname}" ];
+
+  propagatedBuildInputs = [
```

Comment:
I am not sure if this is correct. Are they only required at runtime? Then maybe a wrapper would be a better choice.

Reply:
Yes, they are only required at runtime. I disabled the tests thus those deps are not needed for building.